etc/answers-headless.api.md

```diff
@@ -0,0 +1,670 @@
+## API Report File for "@yext/answers-headless"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Unsubscribe } from '@reduxjs/toolkit';
+
+// @public
+export interface AllResultsForVertical {
+    facets: DisplayableFacet[];
+    results: Result[];
+    resultsCount: number;
+}
+
+// Warning: (ae-forgotten-export) The symbol "AnswersConfigWithApiKey" needs to be exported by the entry point index.d.ts

```

Comment:
Nidhi, can you take a look through this API report and add the exports that API extractors says is missing?  The one exception is the AnswersCore exports because I don't think that needs to be part of the public interface

Reply:
I looked through it, and if those are added, the only things we wouldn't be exporting from Core would be `AnswersCore` and `provideCore()`. (Also, see previous comment on why `AnswersCore` is exported). If that's what we want, to have full documentation for anything we export from Core, maybe there's an easier way to export everything except for one or two things?

Reply:
For now I'd say it's okay to keep the list since we don't want `provideCore` exported. I experimented with `import { * as AnswersCoreExports } from '@yext/answers-core'` and attempting to delete single exports but I couldn't get it working. I don't think we should spend too much time on it now, and we can make a follow up item if needed

Reply:
Updated to include all of the "forgotten exports" from Core